precipitation/ngsild-payloads/Forecast.json

```diff
@@ -0,0 +1,198 @@
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:temperature:CEL:MIN:APIX"
+    },
+    {
+        "type": "Property",
+        "value":20.0,
+        "unitCode":"CEL",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:temperature:CEL:MAX:APIX"
+    },
+    {
+        "type": "Property",
+        "value":15.0,
+        "unitCode":"CEL",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:temperature:CEL:AVG:APIX"
+    }
+    ],
+    "windDirection":[
+        {
+        "type": "Property",
+        "value":220,
+        "unitCode":"DEG",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:direction:DEG:01"
+    }
+],
+    "windSpeed":[    
+        {
+        "type": "Property",
+        "value":50,
+        "unitCode":"KMH",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:Gust:KMH:APIX"
+    },
+    {
+        "type": "Property",
+        "value":20,
+        "unitCode":"KMH",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:wind:KMH:AVG:APIX"
+    }
+],
+    "windProb":[    
+        {
+        "type": "Property",
+        "value":40,
+        "unitCode":"P1",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:prob70:P1:APIX"
+    },
+    {
+        "type": "Property",
+        "value":10,
+        "unitCode":"P1",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:prob100:P1:APIX"
+    }
+],
+    "sunHours":
+    { 
+        "type": "Property",
+        "value":7,
+        "unitCode":"H",
+        "observedAt": "2020-06-26T00:00:00Z",
+        "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:StationY553403001"
+        },
+        "datasetId":"urn:ngsi-ld:Dataset:sunHours:P1:APIX"
```

Comment:
the datasetId have to be used only when you have a table of values (check the temperature for example). So for sunHours you have to  delete the datasetId or to put the value of the property as a table of values with only  one value (the recommended one) as follows : 
```
"sunHours":
[
    { 
        "type": "Property",
        "value":7,
        "unitCode":"H",
        "observedAt": "2020-06-26T00:00:00Z",
        "observedBy": {
            "type": "Relationship",
            "object": "urn:ngsi-ld:Device:StationY553403001"
        },
        "datasetId":"urn:ngsi-ld:Dataset:sunHours:P1:APIX"
     }
]
```
 - urn:ngsi-ld:Dataset:sunHours:P1:APIX  put 'H' instead of 'P1' (the unit of the property).